arbnode/mel/state.go

```diff
@@ -45,6 +45,10 @@ type StateDatabase interface {
 		state *State,
 		delayedMessages []*DelayedInboxMessage,
 	) error
+	DelayedMessageDatabase
```

Comment:
please move that to the top of the interface.. took me a while to figure out what this type is doing here :)